Codeinwp/neve
# Fix LMS colors

### Summary
Seems like Lifter LMS has hardcoded colors in its code, making our dark pallet not working as it is supposed to work.

### Will affect visual aspect of the product
YES

### Test instructions
- Install Lifter LMS and import their courses
- Import a starter site, add the pallet switcher component
- Check if colors are looking ok for both Memberships and Courses
- Check the dashboard page of Lifter
- Check achievements and certificates


Closes Codeinwp/neve-pro-addon#1810.
